s4e-web/src/app/views/map-view/view-manager/layer-picker/layer-picker.component.html

```diff
@@ -6,14 +6,19 @@
 </header>
 <section class="box__content">
   <ul class="list">
-    <li class="list__item" [ngClass]="{active: item.active}" role="menuitem" *ngFor="let item of items" (click)="itemSelected.emit(item.cid)">
-      <figure class="list__icon">
+    <li class="list__item" [ngClass]="{active: item.active}" role="menuitem" *ngFor="let item of items">
+      <ng-container *ngIf="hasFavourite(item)">
+        <img *ngIf="!item.favourite" class="icon__favorite" src="../../../../../assets/images/star-regular.svg" alt="Not favorite icon" i18n-alt />
+        <img *ngIf="item.favourite" class="icon__favorite" src="../../../../../assets/images/star-solid.svg" alt="Favorite icon" i18n-alt />
+      </ng-container>
+
+      <figure class="list__icon" (click)="itemSelected.emit(item.cid)">
         <img src="../../../../../assets/images/ico_product_default.png" alt="Default product icon" i18n-alt />
       </figure>
-      <button class="list__name">
+      <button class="list__name" (click)="itemSelected.emit(item.cid)">
         {{item.caption}}
       </button>
-      <button class="icon__visibility">
+      <button class="icon__visibility" (click)="itemSelected.emit(item.cid)">
         <img src="../../../../../assets/images/ico_eye_open.svg" alt="Eye icon - product visibility" i18n-alt />
```

Comment:
We're generally using Font Awesome in the project. I do recall that stars are part of the FA library. Check for FA usage in the project. (otherwise this is the way assets should be included, but in this particular case it might not be necessary)

Reply:
I was following your code style, that is written beneath lines of mine. In that case, I'll change it.